red/runtime/nodes/flows/index.js

```
@@ -509,8 +628,13 @@ function addFlow(flow) {
             nodes.push(node);
         }
     }
-    var newConfig = clone(activeConfig.flows);
-    newConfig = newConfig.concat(nodes);
+    
+    if (settings.enableFlowSharding) {
+        var newConfig = activeConfig.flows.concat(nodes);
+    } else {
+        var newConfig = clone(activeConfig.flows);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'newConfig'. Consider to delete it or rename it.

'newConfig' is defined here:


(lint/suspicious/noRedeclare)